mirascope/beta/openai/realtime/realtime.py

```diff
@@ -14,6 +14,7 @@
     TypeAlias,
     TypedDict,
     TypeVar,
+    overload,
 )
```

Comment:
also, `NotRequired` needs to be imported from `typing_extensions` for 3.10 support